macro-analytics/README.md

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
Nit: remove blank line at the beginning of the file

Reply:
Thank you, removed!